---
title: "A Brief Introduction on ZooMS"
date: 2023-06-21
author: Fangyuan Sheng
slug: cfa
draft: false
toc: false
tags:
  - English
---
{{<block class="note">}}

I have read several papers on ZooMS, which are truly promising and intriguing. I write a summary on the papers I have read and the photos below credit to *A Primer for ZooMS Applications in Archaeology*.

{{<end>}}

## Terminology

  **ZooMS**: Zooarchaeology by mass spectrometry
  
  **PMF**: Collagen peptide mass fingerprinting
  
  **MALDI-TOF MS**: Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry

  **COL1**: Type I collagen

  **PTM**: Posttranslational modification

## Type I collagen (COL1)

ZooMS can also be referred to as COL1 for taxonomic identification. Before studying ZooMS, we need to know what COL1 is.

COL1 is one of the most ubiquitous and abundant collagen in animals. It can be found in bone, cartilage, antler, ivory, fish scales, and many others.

I draw a simple illustration of bone components.

{{<figure src="https://hellenshengfy.github.io/bone.png">}}


Molecularly, COL1 consists of a **triple helix** made up of three polypeptide α-chains (COL1A). And the composition of triple helix differ across animals. 

  - Tetrapods: two COL1A1 and one COL1A2
  
  - Teleost fish: COL1A1, COL1A2 and COL1A3
  
  - Some others: three COL1A1 

The triple helix consists of amino acids with a repeating G-X-Y motif.

  - G: glycine
    
  - X and Y: both made up of proline and hydroxyproline (a posttranslational modification (PTM) of proline)

Proto-COL1A chains are produced during the formation of COL1 and contain (1) a signal peptide (∼20 amino acids), (2) N-terminal propeptides (∼150 to 300 amino acids), and (3) C-terminal propeptides (∼150 to 300 amino acids). 

The triple helices aggregate to form microfibrils which aggregate to form fibrils, which then bundle together to be collagen fibers. 

{{<figure src="https://hellenshengfy.github.io/zoom1.png">}}

## Why is COL1 important? How does ZooMS work?

Invented in the 1980s, **matrix-assisted laser desorption/ionization (MALDI)** soft ionization method enables large nonvolatile molecules, such as small proteins and peptides, to be ionized without fragmentation for downstream mass spectrometry. 

Later, in the 1990s, **Collagen Peptide Mass Fingerprinting (PMF)** was developed. It helps to identify proteins by the masses of the peptides produced after enzymatic digestion. Coupled with a TOF analyzer, the MALDI-TOF mass spectrometry system is an ideal intrument for PMF analysis. 

It is these two techniques that make ZooMS possible, as the peptides produced after enzymatic digestion of collagen (most commonly COL1) differ in sequence and mass across taxa. See mass spectra below. 

{{<figure src="https://hellenshengfy.github.io/zoom2.png">}}

The different peak positions in the chart above shows the COL1 PMF of different species (turkey (*Meleagris gallopavo*), goat (*Capra hircus*), and coho salmon (*Oncorhynchus kisutch*)). Pink circles represent known collagen peptides that peak in mass. Peptides that are taxonomically informative are called marker peptides. 

We can observe three **marker peptides** presented:

 - COL1A1 508-519
   
 - COL1A2 454-483
   
 - COL1A2 757-789

The turkey, goat, and coho salmon show different PMF patterns and their marker peptides peak at different mass position which are indicative in identification.

## Procedures

(1)  Extract the collagen from matrix

(2)  Digest the collagen with protease (eg:trypsin) to cleave the C-terminal peptide bonds of arginine (R) and lysine (K) residues while adding the mass of a water molecule to each peptide.

(3)  Acidify and purify the peptides with a C18 filter

(4)  Spot peptides onto an MALDI plate with a matrix that cocrystalizes with the peptides. A typical matrix is α-cyano-4-hydroxycinnamic acid. 

(4)  Excite the matirx with a laser to vaporize peptides and ionize them with a +1 charge. 

(5)  Electromagnets direct the ions into a time-of-flight tube. The ions will separates by mass and the smallest peptides will hit the detector first. 


{{<figure src="https://hellenshengfy.github.io/zoom3.png">}}



## Why is ZooMS promising?

(1) Relatively low cost

(2) Low sample input requirement

(3) Robust taxonomic identification

(4) Minimum destruction

 
Reference:

(1) Richter, Kristine Korzow, Maria C. Codlin, Melina Seabrook, and Christina Warinner. “A Primer for ZooMS Applications in Archaeology.” Proceedings of the National Academy of Sciences 119, no. 20 (May 17, 2022): e2109323119. https://doi.org/10.1073/pnas.2109323119. 

(2) M.Buckley etal., Distinguishing betweenarchaeologicalsheepand goatbones using asingle collagenpeptide.J. Archaeol.Sci. 37, 13–20(2010).
